tests/test_config.py

```
@@ -1643,3 +1679,60 @@ def test_dogfood_pin_and_fallback_graph_are_unchanged(tmp_path):
+    global_path = _write(tmp_path / 'global.toml', text)
+    repo = tmp_path / 'repo'
+    repo.mkdir()
+    _write(repo / '.skodun.toml', f'[capacity]\nmachine = {repo_machine}\n')
+    cfg = load_config(repo, global_path=global_path)
+    assert resolved_machine_capacity(cfg, env={'SKODUN_REVIEW_MACHINE_CAPACITY': '4'}) == expected
+
+
+@pytest.mark.parametrize("key", ["_repo_machine", "_repo_review_fg"])
+def test_internal_repo_capacity_metadata_is_not_a_config_key(tmp_path, key):
+    global_path = _write(tmp_path / 'global.toml', f'[capacity]\n{key} = 8\n')
+    with pytest.raises(ValueError, match='unknown.*capacity'):
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)